htmd/builder/charmmfiles/.gitignore

```diff
@@ -1 +0,0 @@
-toppar_*
```

Comment:
add it back with `./toppar_*` and that's all.

Reply:
done